db/migrate/20161004123738_create_grants.rb

```diff
@@ -0,0 +1,12 @@
+class CreateGrants < ActiveRecord::Migration
+  def change
+    create_table :grants do |t|
```

Comment:
Sorry for the incomplete story. Could you please add status. Values would be `Pending` `Active` and `Closed`. It will be required.